src/resources/Activities/Activities.ts

```diff
@@ -0,0 +1,43 @@
+import API from '../../APICore';
+import {PageModel} from '../BaseInterfaces';
+import Indexes from '../Indexes/Indexes';
+import Resource from '../Resource';
+import {ActivityModel, ListActivitiesParams} from './ActivitiesInterfaces';
+
+export default class Activity extends Resource {
+    static getBaseUrl = () => `/rest/organizations/${API.orgPlaceholder}/activities`;
+
+    get(activityId: string) {
+        return this.api.get<ActivityModel>(`${Activity.getBaseUrl()}/${activityId}`);
+    }
+    getResourceTypes() {
+        return this.api.get<ActivityModel>(`${Activity.getBaseUrl()}/resourcetypes`);
+    }
+
+    /**
+     * @deprecated list(params?: ListFieldsParams) is kept for backward compatibility, you should now use `search(params?: FieldListingOptions)`.
+     */
+    list(params?: ListActivitiesParams) {
+        return this.api.post<PageModel<ActivityModel>>(this.buildPath(`${Indexes.baseUrl}/public`, params));
+    }
```

Comment:
Should we really add a deprecated method in the client? 🤔

Reply:
method is not deprecated after all :) my bad